static/app/routes.tsx

```diff
@@ -1313,15 +1313,15 @@ function buildRoutes() {
         }}
       />
       <Route
-        path="tags/"
+        path={TabPaths[Tab.TAGS]}
         component={make(() => import('sentry/views/organizationGroupDetails/groupTags'))}
         props={{
           currentTab: Tab.TAGS,
```

Comment:
Possible to delete these props now?

Reply:
Working on that now

Reply:
But I think so

Reply:
Done! https://github.com/getsentry/sentry/pull/39137/commits/62ef51613bc27579d5373bbeb770540fbdae569b